DWIConvert/DWIConverter.cxx

```diff
@@ -211,7 +201,7 @@ void DWIConverter::ReadGradientInformation(const std::string& inputBValues, cons
   unsigned int                      bVecCount = 0;
   if( ReadBVecs(BVecs, bVecCount, _inputBVectors,true) != EXIT_SUCCESS )
   {
-    itkGenericExceptionMacro(<< "ERROR reading Bvals " << _inputBVectors);
+    itkGenericExceptionMacro(<< "ERROR reading Bvectors " << _inputBVectors);
```

Comment:
Good catch!.

Probably should be BVectors instead of Bvectors to be consistent with naming.